Auto update failing for ccache

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [ccache](https://github.com/termux/termux-packages/blob/604de805839e4b1ee482506e5f0f0ea569ace008/packages/ccache) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating ccache [Current version: 4.9.1]
INFO: package being updated to 4.10.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0  401k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  401k  100  401k    0     0   690k      0 --:--:-- --:--:-- --:--:-- 18.6M

You are about to commit these changes:

--------------------
diff --git a/packages/ccache/build.sh b/packages/ccache/build.sh
index 3a1d7a617..3b9f4d948 100644
--- a/packages/ccache/build.sh
+++ b/packages/ccache/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://ccache.samba.org
 TERMUX_PKG_DESCRIPTION="Compiler cache for fast recompilation of C/C++ code"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="4.9.1"
+TERMUX_PKG_VERSION="4.10"
 TERMUX_PKG_SRCURL=https://github.com/ccache/ccache/releases/download/v$TERMUX_PKG_VERSION/ccache-$TERMUX_PKG_VERSION.tar.xz
-TERMUX_PKG_SHA256=4c03bc840699127d16c3f0e6112e3f40ce6a230d5873daa78c60a59c7ef59d25
+TERMUX_PKG_SHA256=83630b5e922b998ab2538823e0cad962c0f956fad1fcf443dd5288269a069660
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++, zlib, zstd"
 
--------------------

bump(main/ccache): 4.10


Not committing to Git!

INFO: Trying to build package.
INFO: Package ccache exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building ccache for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   235k      0 --:--:-- --:--:-- --:--:--  237k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1281      0 --:--:-- --:--:-- --:--:--  1281
100   833  100   833    0     0   1281      0 --:--:-- --:--:-- --:--:--  1279
gpg: Signature made Wed 29 May 2024 06:22:40 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  24508      0 --:--:-- --:--:-- --:--:-- 24530
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1423      0 --:--:-- --:--:-- --:--:--  1421
100   833  100   833    0     0   1423      0 --:--:-- --:--:-- --:--:--  1421
gpg: Signature made Fri 24 May 2024 06:21:29 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  22441      0 --:--:-- --:--:-- --:--:-- 22465
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1404      0 --:--:-- --:--:-- --:--:--  1407
gpg: Signature made Wed 29 May 2024 06:23:27 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_26b_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  297k  100  297k    0     0  3899k      0 --:--:-- --:--:-- --:--:-- 3915k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zlib/zlib_1.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 60516  100 60516    0     0  1024k      0 --:--:-- --:--:-- --:--:-- 1036k
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.1+really5.4.5 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libl/liblzma/liblzma_5.6.1+really5.4.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  241k  100  241k    0     0  4083k      0 --:--:-- --:--:-- --:--:-- 4090k
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.1+really5.4.5 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/x/xz-utils/xz-utils_5.6.1+really5.4.5_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 63468  100 63468    0     0   518k      0 --:--:-- --:--:-- --:--:--  520k
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zstd@1.5.6 if necessary...
Found zstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/z/zstd/zstd_1.5.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  329k  100  329k    0     0  2681k      0 --:--:-- --:--:-- --:--:-- 2698k
extracting zstd to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pzstd@1.5.6 if necessary...
Found pzstd in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pzstd/pzstd_1.5.6_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 40540  100 40540    0     0   675k      0 --:--:-- --:--:-- --:--:--  682k
extracting pzstd to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/ccache/ccache/releases/download/v4.10/ccache-4.10.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  401k  100  401k    0     0   928k      0 --:--:-- --:--:-- --:--:--  928k
Downloading https://github.com/Kitware/CMake/releases/download/v3.29.3/cmake-3.29.3-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 50.7M  100 50.7M    0     0  71.7M      0 --:--:-- --:--:-- --:--:--  159M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   207k      0 --:--:-- --:--:-- --:--:--  207k
Archive:  /home/builder/.termux-build/ccache/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 17.0.2
-- The CXX compiler identification is Clang 17.0.2
-- The ASM compiler identification is Clang with GNU-like command-line
-- Found assembler: /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang
-- The ASM_MASM compiler identification is unknown
-- Found assembler: ml
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Ccache version: 4.10
-- Ccache dev mode: OFF
-- Performing Test std_atomic_without_libatomic
-- Performing Test std_atomic_without_libatomic - Success
-- Performing Test std_filesystem_without_libfs
-- Performing Test std_filesystem_without_libfs - Success
-- Looking for cpuid.h
-- Looking for cpuid.h - not found
-- Looking for dirent.h
-- Looking for dirent.h - found
-- Looking for linux/fs.h
-- Looking for linux/fs.h - found
-- Looking for pwd.h
-- Looking for pwd.h - found
-- Looking for spawn.h
-- Looking for spawn.h - not found
-- Looking for sys/clonefile.h
-- Looking for sys/clonefile.h - not found
-- Looking for sys/file.h
-- Looking for sys/file.h - found
-- Looking for sys/ioctl.h
-- Looking for sys/ioctl.h - found
-- Looking for sys/mman.h
-- Looking for sys/mman.h - found
-- Looking for sys/sendfile.h
-- Looking for sys/sendfile.h - found
-- Looking for sys/utime.h
-- Looking for sys/utime.h - not found
-- Looking for sys/wait.h
-- Looking for sys/wait.h - found
-- Looking for syslog.h
-- Looking for syslog.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for utime.h
-- Looking for utime.h - found
-- Looking for asctime_r
-- Looking for asctime_r - found
-- Looking for getopt_long
-- Looking for getopt_long - found
-- Looking for getpwuid
-- Looking for getpwuid - found
-- Looking for localtime_r
-- Looking for localtime_r - found
-- Looking for posix_fallocate
-- Looking for posix_fallocate - found
-- Looking for setenv
-- Looking for setenv - found
-- Looking for syslog
-- Looking for syslog - found
-- Looking for unsetenv
-- Looking for unsetenv - found
-- Looking for utimensat
-- Looking for utimensat - found
-- Looking for utimes
-- Looking for utimes - found
-- Performing Test HAVE_STRUCT_STAT_ST_ATIM
-- Performing Test HAVE_STRUCT_STAT_ST_ATIM - Success
-- Performing Test HAVE_STRUCT_STAT_ST_ATIMENSEC
-- Performing Test HAVE_STRUCT_STAT_ST_ATIMENSEC - Success
-- Performing Test HAVE_STRUCT_STAT_ST_ATIMESPEC
-- Performing Test HAVE_STRUCT_STAT_ST_ATIMESPEC - Failed
-- Performing Test HAVE_STRUCT_STAT_ST_CTIM
-- Performing Test HAVE_STRUCT_STAT_ST_CTIM - Success
-- Performing Test HAVE_STRUCT_STAT_ST_CTIMENSEC
-- Performing Test HAVE_STRUCT_STAT_ST_CTIMENSEC - Success
-- Performing Test HAVE_STRUCT_STAT_ST_CTIMESPEC
-- Performing Test HAVE_STRUCT_STAT_ST_CTIMESPEC - Failed
-- Performing Test HAVE_STRUCT_STAT_ST_MTIM
-- Performing Test HAVE_STRUCT_STAT_ST_MTIM - Success
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMENSEC
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMENSEC - Success
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMESPEC
-- Performing Test HAVE_STRUCT_STAT_ST_MTIMESPEC - Failed
-- Performing Test HAVE_STRUCT_STATFS_F_FSTYPENAME
-- Performing Test HAVE_STRUCT_STATFS_F_FSTYPENAME - Failed
-- Performing Test HAVE_AVX2
-- Performing Test HAVE_AVX2 - Failed
-- Using bundled Blake3 since Blake3>=1.4.0 was not found locally
-- Using bundled CppHttplib since CppHttplib>=0.10.6 was not found locally
-- Downloading Fmt from the internet since Fmt>=8.0.0 was not found locally and DEPS=AUTO
-- Version: 10.2.1
-- Build type: Release
-- Performing Test HAS_NULLPTR_WARNING
-- Performing Test HAS_NULLPTR_WARNING - Success
-- Using bundled NonstdSpan since NonstdSpan>=0.10.3 was not found locally
-- Using bundled TlExpected since TlExpected>=1.1.0 was not found locally
-- Downloading Xxhash from the internet since Xxhash>=0.8.0 was not found locally and DEPS=AUTO
-- Using system Zstd (/data/data/com.termux/files/usr/lib/libzstd.so)
-- Downloading doctest from the internet since Doctest>=2.4.6 was not found locally and DEPS=AUTO
-- Performing Test HAVE_C_SSE2
-- Performing Test HAVE_C_SSE2 - Failed
-- Performing Test HAVE_C_SSE41
-- Performing Test HAVE_C_SSE41 - Failed
-- Performing Test HAVE_C_AVX2
-- Performing Test HAVE_C_AVX2 - Failed
-- Performing Test HAVE_C_AVX512
-- Performing Test HAVE_C_AVX512 - Failed
-- Performing Test HAVE_NEON
-- Performing Test HAVE_NEON - Success
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- 
-- Configuration summary:
--   Storage backends:
--     file  ON
--     http  ON
--     redis OFF
--   Dependencies:
--     Blake3     1.5.1  BUNDLED
--     CppHttplib 0.15.3 BUNDLED
--     Doctest    2.4.11 DOWNLOADED
--     Fmt        10.2.1 DOWNLOADED
--     NonstdSpan 0.11.0 BUNDLED
--     TlExpected 1.1.0  BUNDLED
--     Xxhash     0.8.2  DOWNLOADED
--     Zstd       1.5.6  SYSTEM (/data/data/com.termux/files/usr/lib/libzstd.so)
-- 
-- Configuring done (6.2s)
-- Generating done (0.0s)
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


-- Build files have been written to: /home/builder/.termux-build/ccache/build
[1/121] Building CXX object _deps/fmt-build/CMakeFiles/fmt.dir/src/os.cc.o
[2/121] Building C object src/third_party/blake3/CMakeFiles/dep_blake3.dir/blake3/blake3.c.o
[3/121] Building C object CMakeFiles/dep_xxhash.dir/_deps/xxhash-src/xxhash.c.o
[4/121] Building C object src/third_party/blake3/CMakeFiles/dep_blake3.dir/blake3/blake3_dispatch.c.o
[5/121] Building C object src/third_party/blake3/CMakeFiles/dep_blake3.dir/blake3/blake3_portable.c.o
[6/121] Linking C static library libdep_xxhash.a
[7/121] Building C object src/third_party/blake3/CMakeFiles/dep_blake3.dir/blake3/blake3_neon.c.o
[8/121] Linking C static library src/third_party/blake3/libdep_blake3.a
[9/121] Building CXX object src/third_party/cxxurl/CMakeFiles/dep_cxxurl.dir/cxxurl/url.cpp.o
[10/121] Linking CXX static library src/third_party/cxxurl/libdep_cxxurl.a
[11/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/Args.cpp.o
[12/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/Context.cpp.o
[13/121] Building CXX object _deps/fmt-build/CMakeFiles/fmt.dir/src/format.cc.o
[14/121] Linking CXX static library _deps/fmt-build/libfmt.a
[15/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/ProgressBar.cpp.o
[16/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/Depfile.cpp.o
[17/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/Hash.cpp.o
[18/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/Config.cpp.o
[19/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/compopt.cpp.o
[20/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/execute.cpp.o
[21/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/hashutil.cpp.o
FAILED: src/ccache/CMakeFiles/ccache_framework.dir/hashutil.cpp.o 
/home/builder/.termux-build/_cache/android-r26b-api-24-v4/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r26b-api-24-v4 --sysroot=/home/builder/.termux-build/_cache/android-r26b-api-24-v4/sysroot -DXXH_STATIC_LINKING_ONLY -I/home/builder/.termux-build/ccache/build -I/home/builder/.termux-build/ccache/src/src -I/home/builder/.termux-build/ccache/src/src/third_party/blake3 -I/home/builder/.termux-build/ccache/build/_deps/fmt-src/include -isystem /home/builder/.termux-build/ccache/src/src/third_party/cxxurl -isystem /home/builder/.termux-build/ccache/src/src/third_party/nonstd-span -isystem /home/builder/.termux-build/ccache/src/src/third_party/tl-expected -isystem /home/builder/.termux-build/ccache/build/_deps/xxhash-src -isystem /home/builder/.termux-build/ccache/src/src/third_party/cpp-httplib -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -std=c++17 -fPIC -include /home/builder/.termux-build/ccache/build/config.h -Wall -pthread -MD -MT src/ccache/CMakeFiles/ccache_framework.dir/hashutil.cpp.o -MF src/ccache/CMakeFiles/ccache_framework.dir/hashutil.cpp.o.d -o src/ccache/CMakeFiles/ccache_framework.dir/hashutil.cpp.o -c /home/builder/.termux-build/ccache/src/src/ccache/hashutil.cpp
/home/builder/.termux-build/ccache/src/src/ccache/hashutil.cpp:449:3: error: unknown type name 'posix_spawn_file_actions_t'
  posix_spawn_file_actions_t file_actions;
  ^
1 error generated.
[22/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/argprocessing.cpp.o
[23/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/language.cpp.o
[24/121] Building CXX object src/ccache/CMakeFiles/ccache_framework.dir/ccache.cpp.o
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-05-30 00:35:20 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/9294862834">9294862834</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/ccache/build.sh
+++ b/packages/ccache/build.sh
@@ -2,18 +2,24 @@ TERMUX_PKG_HOMEPAGE=https://ccache.samba.org
 TERMUX_PKG_DESCRIPTION="Compiler cache for fast recompilation of C/C++ code"
 TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="4.9.1"
+TERMUX_PKG_VERSION="4.10"
 TERMUX_PKG_SRCURL=https://github.com/ccache/ccache/releases/download/v$TERMUX_PKG_VERSION/ccache-$TERMUX_PKG_VERSION.tar.xz
-TERMUX_PKG_SHA256=4c03bc840699127d16c3f0e6112e3f40ce6a230d5873daa78c60a59c7ef59d25
+TERMUX_PKG_SHA256=83630b5e922b998ab2538823e0cad962c0f956fad1fcf443dd5288269a069660
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libc++, zlib, zstd"
+TERMUX_PKG_DEPENDS="libandroid-spawn, libc++, libhiredis, xxhash, zlib, zstd"
+TERMUX_PKG_BUILD_DEPENDS="fmt"
 
 #[46/89] Building ASM object src/third_party/blake3/CMakeFiles/blake3.dir/blake3_sse2_x86-64_unix.S.o
 #FAILED: src/third_party/blake3/CMakeFiles/blake3.dir/blake3_sse2_x86-64_unix.S.o
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DDEPS=LOCAL
+-DENABLE_TESTING=OFF
 -DHAVE_ASM_AVX2=FALSE
 -DHAVE_ASM_AVX512=FALSE
 -DHAVE_ASM_SSE2=FALSE
 -DHAVE_ASM_SSE41=FALSE
--DREDIS_STORAGE_BACKEND=OFF
 "
+
+termux_step_pre_configure() {
+	LDFLAGS+=" -landroid-spawn"
+}

